<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>108</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>108</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>108</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add path</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/images/builder.png</normaloff>:/new/images/builder.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>PATH:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>71</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>71</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pathButton">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>31</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>31</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
